views/index.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Welcome to Chitter!</h1>
+<br>
+<form action="/peeps" method="post">
+  <input type="text" value="enter peep here" name="enter peep">
+  <input type="submit" value="Peep"
```

Comment:
My editor highlights </form> as causing a syntax error. That prevents the submit button from doing anything. Turns out you are missing a `>` on line 5 - now the button submits.

Reply:
Indeed! Spotted that yesterday morning and changed it. Just haven't pushed code yet. Thanks.

Reply:
Code has now been pushed :)